current-events/activity-current-events.md

```diff
@@ -0,0 +1,160 @@
+
+4. Within the discussion groups, take turns so that every person introduces their current event topic with their own slide deck.
+
+5. After everyone has presented their current event topic, pick a current event to begin the discussion. Then, work together as a team to discuss the following:
+
+    - How does this news make you feel? Do you feel:
+        - celebratory?
+        - encouraged?
+        - angry?
+        - happy or pleased?
+        - sad?
+        - curious?
+        - amazed?
+        - hungry?
+    - Who are the people most involved or affected by this?
+    - Does this affect me? Why or why not?
+    - How do we learn more?
+
+6. When the conversation has come to a stop, move to the next topic.
+
+7. Congratulate you and your team for engaging in critical discussion! This requires openness, vulnerability, and effort.
+
+## Slide Deck Requirements
+
+- Your name and pronouns
+- A link to the article/piece of media
+- A description of the contents of the article/piece of media
+- A graphic/photograph/picture/logo
+- Answer this question: Why did you think it was interesting?
+- A content warning slide listing all content warnings if relevant (details below)
+
+### !callout-info
+
+## Separation From Current Events
+
+The current news we bring and discuss is not necessarily a reflection of who we are.
+
+### !end-callout
+
+### !callout-success
+
+## Keep Each Other Safe. Keep Yourself Safe.
+
+The safety of marginalized identities in tech is the priority of this activity.
+
+### !end-callout
+
+### !callout-info
+
+## Getting Started
+
+If you're unsure where to begin research, we can begin with finding recent news any of these topics:
+
+- Successful advancements/achievements in science and technology
+- Net neutrality
+- New combinations of new media, art, performance, and technology
+- Working conditions in tech
+- New findings in space exploration
+- Publicized security breaches
+
+### !end-callout
+
+## Content Warning Requirement
+
+Partaking in conversations about current events often means engaging with material that can trigger a strong or potentially harmful emotional response in you or someone else.
+
+<details>
+
+<summary>Create a first, initial slide that lists the content warnings if there's mention of any of the following. This slide should be titled "Content Warnings," and then clearly list the relevant content warnings.</summary>
+
+- Rape and Sexual Assault
+- Abuse (physical, mental, emotional, verbal, sexual)
+- Child abuse/pedophilia
+- Animal cruelty or animal death
+- Self-injurious behavior (self-harm, eating disorders, etc.)
+- Suicide
+- Excessive or gratuitous violence
+- Needles
+- Depiction of pornography (including child pornography)
+- Incest (including any and all elements of romantic or sexual relationships between family, tonal in theme, thought, or activity)
+- Kidnapping (forceful deprivation of/disregard for personal autonomy)
+- Miscarriages/Abortion
```

Comment:
capitalization and punctuation
```suggestion
- Rape or sexual assault
- Abuse (physical, mental, emotional, verbal, sexual, etc.)
- Child abuse or pedophilia
- Animal cruelty or animal death
- Self-injurious behavior (self-harm, eating disorders, etc.)
- Suicide
- Excessive or gratuitous violence
- Needles
- Depiction of pornography (including child pornography)
- Incest (including any and all elements of romantic or sexual relationships between family, tonal in theme, thought, or activity)
- Kidnapping (forceful deprivation of, or disregard for, personal autonomy)
- Miscarriages or abortion
```

Reply:
I'm manually editing your suggestion to remove the "etc." from the list. I think it's more effective in this case to have a definitive list, as I feel that the "etc." opens it up for more interpretation.